include/kalmar_runtime.h

```diff
@@ -922,6 +923,10 @@ struct rw_info
         /// 2. Because the data pointer cannout be released, erase itself from devs
         if (HostPtr)
             synchronize(false);
+        if (curr) {
+            // Wait issues a system-scope release:
+            curr->wait();
```

Comment:
@bensander this is the most important change to tame `array_view`-based tests, right?